lib/board_selection.py

```diff
@@ -0,0 +1,15 @@
+from .tictactoe_board import TicTacToeBoard
+
+
+class BoardSelection:
+    def __init__(self, ui):
+        self.ui = ui
+
+    def select_board_size(self):
+        self.ui.display("Welcome to TIC TAC TOE!")
+        self.ui.display("First, select a board size.")
+        size = int(self.ui.get_user_input())
+        if size != 3 and size != 4:
+            self.ui.display("The size of the board should be 3 or 4, try again!")
+            size = int(self.ui.get_user_input())
```

Comment:
A user is able to bypass entering an invalid value if they enter an invalid value twice (ex. `0`, `0`)

Reply:
Done!